cluster/images/conformance/run_e2e.sh

```diff
@@ -36,6 +36,14 @@ saveResults() {
     echo -n "${RESULTS_DIR}/e2e.tar.gz" > "${RESULTS_DIR}/done"
 }
 
+# Optional Golang runner alternative to the bash script.
+# Entry provided via env var to simplify invocation. 
+if [[ -n ${E2E_USE_GO_RUNNER:-} ]]; then
```

Comment:
I'm not exactly sure I understand the question; if I push a commit where it is the default (just for testing) it should run fine; the log and junit output is the same.  The problem I'm having right now it seems though is build related; maybe you can help.

It seems like `pull-kubernetes-conformance-image-test` is calling into the `test-infra` repo which doesn't build `go-runner` properly so it messes things up I think. I must be missing some part of how to integrate this into the build. The error is below but the test infra code is here (https://github.com/kubernetes/test-infra/blob/master/experiment/kind-conformance-e2e.sh#L26)

```
W0622 14:50:22.002] cp: cannot stat '/go/src/k8s.io/kubernetes/cluster/images/conformance/../../../_output/dockerized/bin/linux/amd64/go-runner': No such file or directory
...
W0622 14:50:28.225] subprocess.CalledProcessError: Command '('bash', '-c', 'cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh')' returned non-zero exit status 2
E0622 14:50:28.230] Command failed
```

So I've integrated into the build and as a test target, is there somewhere else I need to update? Or is there a different place just for the dockerized artifacts? Or is this a problem because of a disconnect between test infra and k/k?

Reply:
forgot that our script:
https://github.com/kubernetes/test-infra/blob/master/experiment/kind-conformance-e2e.sh#L40
loads an image from GCR and does not build a new image, thus the runner will be absent there.

so my idea will not work.

Reply:
That's just a `kind load` which is going to load the image you have tagged locally, right? And right above that, it builds this image.

I think that's the problem I'm having now: It tries to build/load the image but that script is in the `test-infra` repo. @fejta said there isn't a way to update that script at the same time as this PR because they are in different repos; so I feel like I'm in a chicken/egg situation.